src/ui/actions/metadata.ts

```diff
@@ -3,10 +3,10 @@ import { Action } from "redux";
 import { selectors } from "ui/reducers";
 import { UIStore, UIThunkAction } from ".";
 import { Comment } from "ui/state/metadata";
-import { prefs, features } from "ui/utils/prefs";
+const { prefs, features } = require("ui/utils/prefs");
 import { seek } from "./timeline";
 import { User } from "ui/state/metadata";
-import LogRocket from "ui/utils/logrocket";
```

Comment:
the impetus for using import was that Logrocket has an `export default` we could either tack a default at the end `require("ui/utils/logrocket").default;` or add a type definition

Reply:
OK, got it. I've added `default` to the end of the `require`.